connexion/resolver.py

```diff
@@ -45,7 +46,7 @@ def resolve_operation_id(self, operation):
         :type operation: connexion.operation.Operation
         """
         spec = operation.operation
-        operation_id = spec.get('operationId')
+        operation_id = spec.get('operationId', '')
```

Comment:
This will show error that empty module not found. Which in my opinion is better for debugging.

Removing Connexion from de way when `operationId` is not defined in the specification file. This gives a hint to developers that a module must exist in the  `operationId` field of the specification.